pkg/transform/snowplow_gtmss_preview_test.go

```diff
@@ -244,6 +244,29 @@ func TestGTMSSPreview(t *testing.T) {
 			ExpInterState: nil,
 			Error:         errors.New("Message has expired"),
 		},
+		{
+			Scenario:  "no_gtmss_no_expire",
```

Comment:
`main_case_no_gtmss` should already cover this case - but the expiry should be shorter. :)

Reply:
Maybe keep both? One using fifty years, the other with shorter timeout (1 hour)

Reply:
What's are we testing with 50 year expiry config, and no gtm ss header?

Reply:
That....it doesn't go to bad? You can't never be sure when I'm an author of some code